### PR TITLE
Full app navigation

### PR DESCRIPTION
**Pull request by dwalintukan**
- Added back button
- Fixed showing nav arrow in detail pages

image

### CODE REVIEW

File: src/scenes/Activities/index.js [hunk 1]
```diff
@@ -84,27 +120,36 @@ class Activities extends React.Component {
   }
 
   handleTabChange(event, value) {
-    this.setState({ tabIdx: value });
+    switch (value) {
+      case 0: {
```
Comment (liviavinci) [inline src/scenes/Activities/index.js]: Can we use a const to store the tab index value?

File: src/scenes/Activities/index.js [hunk 2]
```diff
@@ -30,11 +30,47 @@ const messages = defineMessages({
...
+  };
+
   constructor(props) {
     super(props);
 
+    // Determine tab index based on path
+    let tabIdx;
+    switch (this.props.match.path) {
+      case RouterPath.set: {
+        tabIdx = 0;
```
Comment (liviavinci) [inline src/scenes/Activities/index.js]: Can we use a const to store the tab index value?

File: src/scenes/Activities/index.js (hunk 1 above)
Comment (dwalintukan) [reply src/scenes/Activities/index.js]: good idea 👍

File: src/scenes/Activities/index.js (hunk 2 above)
Comment (dwalintukan) [reply src/scenes/Activities/index.js]: done

File: src/components/BackButton/index.jsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+import styles from './styles';
+import { AppLocation, RouterPath } from '../../constants';
+
+@withRouter
+@injectIntl
+@withStyles(styles, { withTheme: true })
+@connect((state, props) => ({
+  appLocation: state.App.get('appLocation'),
+}), (dispatch, props) => ({
```
Comment (alex-cory) [inline src/components/BackButton/index.jsx]: @dwalintukan if we're not using the dispatch, we don't need to put it here :)

File: src/components/NavBar/index.js [hunk 4]
```diff
@@ -89,50 +110,54 @@ class NavBar extends React.PureComponent {
...
-
-    return null;
+    return (
+      <img
+        src="/images/nav-arrow.png"
+        alt="nav-arrow"
+        className={
+          classNames(
+            classes.navArrow,
+            currentPath === RouterPath.myWallet || currentPath === RouterPath.set ? 'right' : ''
```
Comment (alex-cory) [inline src/components/NavBar/index.js]: there's a better way to do this. @dwalintukan 
```
classNames(classes.navArrow, {
  right: currentPath === RouterPath.myWallet || currentPath === RouterPath.set
})
```

File: src/constants.js [hunk 5]
```diff
@@ -1,10 +1,33 @@
 module.exports = {
+  RouterPath: {
```
Comment (alex-cory) [inline src/constants.js]: do we use these anywhere else? I personally find it more confusing to have to go to this file and see what is going on than explicitly putting this where it's being used. @dwalintukan

File: src/components/BackButton/index.jsx [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+}))
+export default class BackButton extends React.Component {
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+    history: PropTypes.object.isRequired,
+    appLocation: PropTypes.string.isRequired,
+  };
+
+  render() {
+    const {
```
Comment (alex-cory) [inline src/components/BackButton/index.jsx]: can we please put these on one line unless they're extremely long? Our files just keep getting longer and longer from small things like this <3 @dwalintukan

File: src/components/BackButton/index.jsx (hunk 3 above)
Comment (dwalintukan) [reply src/components/BackButton/index.jsx]: gotcha, will remove in a later cleanup PR

Comment (dwalintukan) [reply src/components/BackButton/index.jsx]: so just dont include the 2nd param?

File: src/components/BackButton/index.jsx (hunk 6 above)
Comment (dwalintukan) [reply src/components/BackButton/index.jsx]: sure will remember for the future ones. this one already merged in :P

File: src/components/NavBar/index.js (hunk 4 above)
Comment (dwalintukan) [reply src/components/NavBar/index.js]: so the { } will return undefined? or will it be `right: undefined`?

**Pull request merged**